apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx

```diff
@@ -31,12 +31,16 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
 			<AppAvatar size='x124' mie='x20' iconFileContent={iconFileContent} iconFileData={iconFileData} />
 			<Box display='flex' flexDirection='column'>
 				<Box display='flex' flexDirection='row' alignItems='center' mbe='x8'>
-					<Box fontScale='h1' mie='x8'>
+					<Box fontScale='h1' mie='x8' color='default'>
```

Comment:
@felipe-rod123 changing this to default color means that we also change it on the light theme? 🤔

Reply:
Yes, as far as I know, the new color system will adapt to the light and dark theme automatically